package.json

```diff
@@ -105,6 +108,9 @@
     "terra-icon": "^2.0.1",
     "terra-image": "^2.0.1",
     "terra-markdown": "^2.0.0",
+    "terra-menu": "^2.0.1",
+    "terra-navigation-layout": "^2.0.0",
```

Comment:
We should be able to drop `terra-menu` and `terra-navigation-layout`. In this PR #61, Matt updated the dev-site to use `terra-application-layout` which provides these pieces for us.